repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,147 @@
+{
+  "type": "object",
+  "properties": {
+    "marathon/libprocess-ip": {
```

Comment:
That would be quite confusing in my opinion, since it's not the interface we necessarily bind on for the API service.